internal/use_cases/create_pull_request.go

```diff
@@ -324,3 +324,12 @@ func (cpr *CreatePullRequest) createNewUserBranchAndPush(baseBranch string, issu
 
 	return branchName, false, nil
 }
+
+func (cpr *CreatePullRequest) getIssueTypeLabel(issue domain.Issue) (string, error) {
+	issueTracker, err := cpr.IssueTrackerProvider.GetIssueTracker(issue.ID)
+	if err != nil {
+		return "", err
+	}
+
+	return issueTracker.GetIssueTypeLabel(issue)
+}
```

Comment:
I think you are forced to get this again because of if-else at the beginning of the funcion. We should try to refactor that part and just get the issueTracker once.